tests/all-tests.html

```diff
@@ -72,13 +52,7 @@
         "./component-tests/tabs/html/Tabs-test.html",
         "./component-tests/tooltip/html/Tooltip-test.html",
         "./component-tests/uiOptions/html/UIOptions-test.html",
-        "./component-tests/uploader/html/DemoUploadManager-test.html",
-        "./component-tests/uploader/html/ErrorPanel-test.html",
-        "./component-tests/uploader/html/FileQueue-test.html",
-        "./component-tests/uploader/html/FileQueueView-test.html",
-        "./component-tests/uploader/html/HTML5UploaderSupport-test.html",
-        "./component-tests/uploader/html/Uploader-test.html",
-        "./component-tests/uploader/html/UploaderCompatibility-test.html",
+        "./component-tests/uploader/all-tests.html",
```

Comment:
Yes, I would prefer we didn't do this, just on general principles. If we did, we would at least need to make sure to validate it with @avtar 's integration server (at least, that it didn't result in more test failures than we already have). The cost of cut'n paste maintenance here I think is small compared to the risks, or at least our current apprehension of them